spec/features/user_edits_location_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+feature 'user edits a location', %Q{
+  As a signed in user
+  I want to edit a location
+  So that I can update the locations database
+} do
+
+  scenario 'signed in user edits a location successfully' do
+    user = User.create(
+      first_name: "Boris",
+      last_name: "Johnson",
+      email: "jsmith@email.com",
+      password: "password")
+
+    location = Location.create(
+      name: "Top of the state",
+      address: "123 address st",
+      city: "Boston",
+      state: "MA",
+      zip: "02112",
+      user_id: user.id
+    )
+
+    visit new_user_session_path
```

Comment:
you shouldnt need `visit new_user_session_path` if you are using the `sign_in_as(user)` helper